website/templates/project/retract_registration.mako

```diff
@@ -0,0 +1,49 @@
+<%inherit file="project/project_base.mako"/>
+<%def name="title()">Retract Registration of Component</%def>
+
+<legend class="text-center">Retract Registration</legend>
+
+    <div id="registrationRetraction" class="col-md-6 col-md-offset-3">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                Retracting a registration will remove its content from the OSF, but leave basic meta-data behind.
+                The title of a retracted registration and its contributor list will remain, as will justification or
+                explanation of the retraction, should you wish to provide it. Retracted registrations will be marked
+                with a "retracted" tag. <strong>This action is irreversible.</strong>
+            </div>
+        </div>
+        <form id="registration_retraction_form" role="form">
```

Comment:
> Use .lowercase-and-dashes for class names and #camelCase for IDs.

https://cosdev.readthedocs.org/en/latest/style_guides/html_css.html#html-and-css